<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QueryBookDialog</class>
 <widget class="QDialog" name="QueryBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查询图书信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="bookIDLabel">
       <property name="text">
        <string>图书编号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="bookIDLineEdit"/>
     </item>
     <!-- 合并书名和作者为关键词搜索 -->
     <item row="1" column="0">
      <widget class="QLabel" name="keywordLabel">
       <property name="text">
        <string>关键词：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="keywordLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="borrowedStatusLabel">
       <property name="text">
        <string>借阅状态：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="borrowedStatusComboBox">
       <item>
        <property name="text">
         <string>全部</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>已借出</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>可借</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="searchButton">
     <property name="text">
      <string>搜索</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="resultTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
